Doc update (#171)
ufs-community/ufs-mrweather-app: docs/UsersGuide/source/introduction.rst

The MR Weather App is distributed with the :term:`chgres_cube` pre-processing software.
It converts the Global Forecast System (GFS) analyses to the format needed as
input to the model, which is six tiles in NetCDF format. Additional information

> about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/latest/>`_.

Reviewer: ```suggestion
about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/ufs-v1.1.0/>`_.
```

Author: Changed.

GFS analyses for initializing the MR Weather App can be in Gridded Binary
v2 (:term:`GRIB2`) format (in 0.25 , 0.50, or 1.0 degree grid spacing),  the NOAA Environmental

> Modeling System (:term:`NEMS`) Input/Output (:term:`NEMSIO`) format, or Network Common Data Formt (:term:`NETCDF`).

Author: Changed, thanks!

Initialization from dates starting on January 1, 2018 are supported. Dates
before that may work, but are not guaranteed. GFS public archives can be

> accessed through the National Centers for Environmental Information (NCEI)
> `Global Forecast System website <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_
> or through the `Thredds Data Server at NCEI <https://www.ncei.noaa.gov/thredds/model/gfs.html>`_. Some NETCDF data can be found at `the EMC UFS ftp site <https://ftp.emc.ncep.noaa.gov/EIB/UFS/>`_.

Reviewer: ```suggestion
accessed through the `THREDDS Data Server at NCEI <https://www.ncei.noaa.gov/thredds/model/gfs.html>`_. A small sample of netCDF files can be found at `the EMC FTP site <https://ftp.emc.ncep.noaa.gov/EIB/UFS/>`_.
```

Author: changed

core is at `here <https://noaa-emc.github.io/FV3_Dycore_ufs-v1.0.0/html/index.html>`_.
Interoperable atmospheric physics, along with the Noah land surface model, are
supported through the use of the Common Community Physics Package (:term:`CCPP`;
described `here <https://dtcenter.org/community-code/common-community-physics-package-ccpp>`_).
There are four physics suites supported for the release.
Two of them are variations of an updated version of the physics :term:`suite` used in
the operational GFS v15, while the other two are variations of an experimental
suite that includes a subset of the developments for the next version of GFS,
GFS v16. The variations pertain to how the sea surface temperature (SST) is
initialized and parameterized to evolve, and are chosen depending on the type
of initial conditions for the App. Initial conditions in :term:`GRIB2` format have a
single two-dimensional field to initialize the SST, which must be kept constant

> throughout the forecast. Initial conditions in :term:`NEMSIO` or :term:`NETCDF` format have two two-dimensional

Author: Changed.

The MR Weather App is distributed with a post-processing tools, the Unified
Post Processor (UPP). The Unified Post Processor (UPP) converts the

> native NetCDF output from the model to the :term:`GRIB2` format on standard isobaric

Author: Changed.

> The scripts are available in the ftp site ftp://ftp.emc.ncep.noaa.gov/EIB/UFS/visualization_example/.

Reviewer: ```suggestion
The scripts are available in the FTP site ftp://ftp.emc.ncep.noaa.gov/EIB/UFS/visualization_example/.
```

Author: Changed, thanks!

File visualization_README describes the plotting scripts. Example plots are provided

> for the C96 5-day forecasts initialized on 8/29/2019 00 UTC using :term:`GRIB2`,  :term:`NEMSIO`, or :term:`NETCDF`

Author: Changed.
